test/e2e/specs/editor/various/post-title.spec.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post title', () => {
+	test.describe( 'Focus handling', () => {
+		test( 'should focus on the post title field when creating a new post in visual mode', async ( {
+			editor,
+			admin,
+		} ) => {
+			await admin.createNewPost();
+
+			const pageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			await expect( pageTitleField ).toBeFocused();
+		} );
+
+		// Known bug. Will need TextareaControl to expose a ref for this to
+		// work.
+		// eslint-disable-next-line playwright/no-skipped-test
+		test.skip( 'should focus on the post title field when creating a new post in code editor mode', async ( {
```

Comment:
@ciampo I have had to skip this test because `TextareaControl` doesn't expose any API for managing focus. Is that intentional? Did I miss a way to do this?

Could expose just the focus management via `useImperativeHandle`?

Reply:
This seems like a small thing but this will cause an a11y regression because the title field will no longer have auto-focus in Code Editor view mode.